### PR TITLE
dropdown menu

### PR DESCRIPTION
**Pull request by danigallegdup**
Here is the basics for your home page :)

### CODE REVIEW

File: src/main/webapp/home-page/index.html [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+<head>
+  <meta charset="UTF-8">
+  <title>My Portfolio</title>
```
Comment (kieranyyu) [inline src/main/webapp/home-page/index.html:5]: nit: Could set a title, like "Home"?

**Comment by kieranyyu** [comment]
The dropdown looks really nice!
Note: edited the pull request to merge into the "development" branch instead of "main"

**Comment by marcusblake** [comment]
nit: Can you add a screenshot of the homepage as a PR comment?

**Review by marcusblake — DISMISSED** [review]
Great work!

File: src/main/webapp/home-page/index.html (hunk 1 above)
Comment (mingyi09) [inline src/main/webapp/home-page/index.html:5]: Perhaps set a title, header, or short description of our web?

**Review by mingyi09 — COMMENTED** [review]
Looks good!

File: src/main/webapp/home-page/index.html (hunk 1 above)
Comment (danigallegdup) [reply src/main/webapp/home-page/index.html:5]: will do!

**Review by kieranyyu — DISMISSED** [review]

**Review by marcusblake — DISMISSED** [review]

**Comment by marcusblake** [comment]
@danigallegdup do you mind resolving the merge conflict and merging this PR?

**Review by marcusblake — APPROVED** [review]

**Comment by kieranyyu** [comment]
I'm not sure if this is correct, but wouldn't merging this branch remove the script.js and styles.css files from the event-info-page folder? (From looking at "Files changed")

**Comment by marcusblake** [comment]
> I'm not sure if this is correct, but wouldn't merging this branch remove the script.js and styles.css files from the event-info-page folder? (From looking at "Files changed")

That is a good catch @katherine-yu . @danigallegdup it looks like you merged the changes from main. Reach out to me when you get the chance so that we can work on fixing this!

**Pull request closed without merging**